notebooks/tutorials/1_getting_started_lesson4/notebook.ipynb

```diff
@@ -0,0 +1,406 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Pipeline imports\n",
+    "from quantopian.pipeline import Pipeline\n",
+    "from quantopian.pipeline.data.psychsignal import stocktwits\n",
+    "from quantopian.pipeline.factors import SimpleMovingAverage\n",
+    "from quantopian.pipeline.experimental import QTradableStocksUS\n",
+    "\n",
+    "\n",
+    "# Pipeline definition\n",
+    "def  make_pipeline():\n",
+    "\n",
+    "    base_universe = QTradableStocksUS()\n",
+    "\n",
+    "    sentiment_score = SimpleMovingAverage(\n",
+    "        inputs=[stocktwits.bull_minus_bear],\n",
+    "        window_length=3,\n",
+    "    )\n",
+    "\n",
+    "    return Pipeline(\n",
+    "        columns={\n",
+    "            'sentiment_score': sentiment_score,\n",
+    "        },\n",
+    "        screen=base_universe\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For simplicity, we will only analyze the top 350 and bottom 350 stocks ranked by `sentiment_score`. We can create pipeline filters for these sets using the `top` and `bottom` methods of our `sentiment_score` output, and combine them using the `|` operator to get their union. Then, we will remove anything outside of our tradable universe by using the `&` operator to get the intersection between our filter and our universe:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Pipeline imports\n",
+    "from quantopian.pipeline import Pipeline\n",
+    "from quantopian.pipeline.data.psychsignal import stocktwits\n",
+    "from quantopian.pipeline.factors import SimpleMovingAverage\n",
+    "from quantopian.pipeline.experimental import QTradableStocksUS\n",
+    "\n",
+    "# Pipeline definition\n",
+    "def  make_pipeline():\n",
+    "\n",
+    "    base_universe = QTradableStocksUS()\n",
+    "\n",
+    "    sentiment_score = SimpleMovingAverage(\n",
+    "        inputs=[stocktwits.bull_minus_bear],\n",
+    "        window_length=3,\n",
+    "    )\n",
+    "\n",
+    "    # Create filter for top 350 and bottom 350\n",
+    "    # assets based on their sentiment scores\n",
+    "    top_bottom_scores = (\n",
+    "        sentiment_score.top(350) | sentiment_score.bottom(350)\n",
+    "    )\n",
+    "\n",
+    "    return Pipeline(\n",
+    "        columns={\n",
+    "            'sentiment_score': sentiment_score,\n",
+    "        },\n",
+    "        # Set screen as the intersection between our filter\n",
+    "        # and trading universe\n",
+    "        screen=(\n",
+    "            base_universe\n",
+    "            & top_bottom_scores\n",
```

Comment:
I would define `our_universe = base_universe & top_bottom_scores` slightly above this and set `screen=our_universe`, just to be even more clear what's happening.